x/commitment/types/types.go

```diff
@@ -1 +1,10 @@
 package types
+
+// int mapping
+const (
+	ALL_PROGRAM   int64 = 0
+	USDC_PROGRAM        = 1
+	ELYS_PROGRAM        = 2
+	EDEN_PROGRAM        = 3
+	EDENB_PROGRAM       = 4
```

Comment:
Adding Enum on protobuf would be better for autocompile rather than hard typing for all off the enum info

Reply:
Okay agree

Reply:
fixed